CHANGES.rst

```diff
@@ -1,6 +1,27 @@
 Release Notes
 =============
 
+2.1.20
+------
+
+This release improves interpreter discovery to prefer more recent patch versions, e.g. preferring
+Python 3.6.10 over 3.6.8.
```

Comment:
idk if I should add "and preferring both over 3.7.*". I think that's too much detail.

Reply:
Yeah this is fine.